tools/topology/topology2/include/components/pipeline.conf

```diff
@@ -57,7 +57,7 @@ Class.Widget."pipeline" {
 		}
 	}
 
-	# pipeline priority
+	# pipeline priority (0 = highest priority)
```

Comment:
can we add more? like priority range: 0 ~ ?

Reply:
ok, yeah, also useful to add "lowest = 7" iirc.

Reply:
@btian1 @lgirdwood I agree but this is complex. 0 = highest applies to all implementations, but fine details of other semantics are still a  bit unclear to me, including what is the lowest. in IPC4 this is 7, but we don't have such for IPC3. but fair enough, let me investigate more to come up with the documentation.